website/pages/en/substreams.mdx

```diff
@@ -0,0 +1,42 @@
+---
+title: Substreams
+---
+
+Substreams is a powerful blockchain indexing technology, developed for The Graph Network. Substreams enables developers to write Rust modules, composing data streams alongside the community, and provides extremely high-performance indexing by virtue of parallelization, in a streaming-first fashion.
+
+With Substreams, you can extract data from different blockchains (Ethereum, BNB, Solana...) ultra-fast! Then, you can send to data to several locations (a Postgres database, a Mongo database, or a Subgraph).
+
+## How Substreams Works in 4 Steps
+
+1. **You write a Rust program, which defines the transformations that you want to apply to the blockchain data.** For example, the following Rust function extracts relevant information from an Ethereum block (number, hash, and parent hash).
+
+```rust
+fn map_block_meta(blk: Block) -> Result<BlockMeta, substreams::errors::Error> {
+    let header = blk.header.as_ref().unwrap();
+
+    Ok(BlockMeta {
+        number: blk.number,
+        hash: Hex::encode(&blk.hash),
+        parent_hash: Hex::encode(&header.parent_hash),
+    })
+}
+```
+
+2. **You wrap up your Rust program into a WASM container.** Don't worry if it sounds too complex! You can generate the WASM container just by executing a CLI command.
+
+3. **The WASM container is sent to a Substreams provider (StreamingFast or Pinax, for example) for execution.** The Substreams provider feeds the WASM container with the blockchain data and the trasformations are applied.
```

Comment:
Substreams endpoint (from SF or Pinax, for example) ?

Reply:
let's remove the references to individual providers, until we have that reachable through the decentralized network and/or the studio